syne_tune/optimizer/schedulers/searchers/bayesopt/gpautograd/gluon_blocks_helpers.py

```diff
@@ -103,7 +103,7 @@ class ScalarEncodingBase:
     ``dec`` is used in :meth:`set`. Use class:'IdentityScalarEncoding' for no
     encoding (identity). Note: ``enc`` (and ``dec``) must be strictly increasing.
 
-    Box constraints are given by ``constr_lower_int < constr_upper_int``.
+    Box constraints are given by ``constr_lower_int <= constr_upper_int``.
```

Comment:
Should have been fixed earlier. Here, I need it to fix lambda at the beginning and only optimize it when there is enough data